chapters/introduction.tex

```diff
@@ -53,6 +53,7 @@ \section{Scope of the Specification}\label{scope-of-the-specification}
 Initial setup of the model is specified using \lstinline!start!-attributes and equations that hold only during initialization.
 
 A Modelica class may also contain annotations, i.e., formal comments, which specify graphical representations of the class (icon and diagram), documentation text for the class, and version information.
+See \cref{annotations} and note that this uses a special syntax according to \cref{how-to-interpret-annotation-definitions}.
```

Comment:
Doesn't this belong in `\cref{notation}`?

Reply:
I thought adding a link from the introduction was enough, and similarly as the synchronous chapter defines its own notations I think the annotation chapter can do the same.

Reply:
I'm not saying that a link isn't enough; I'm saying that the link belongs in the `\cref{notation}` section so that it can be found by someone looking for an explanation of the notation.

Reply:
OK.  Resolving.